### DISCUSSION TITLE
Need button to copy repo address for cloning

### DISCUSSION THREAD

**Discussion by neonflame4** [community/community#5912] [Mobile]
In the web version, there's a simple button to copy the address repo address for SSH or HTTP cloning.  I'm not able to find this in the app.

**Comment by brianlovin** [comment]
Hey @neonflame4 thanks for the note — are you trying to clone a repo on your phone, or you just need the URL for some other reason?

**Comment by neonflame4** [comment]
Hi Brian,

I have been experimenting with using the iPhone and iPad for quick updates to web projects, which I edit via various text edit apps.

When setting up new projects in the IDE or text editor app, I automatically go to the GitHub app thinking I can quickly grab the repo URL in SSH or HTTPS flavors.  There is no place to do this without going to the GitHub site.  

Let's assume it's a private repo and I'm not logged in to the site (despite being logged into the app).  I'll get a 404 page until I authenticate. There's just a lot of extra steps that could be simplified by allowing the user to copy the URLs out of the app.

**Reply by brianlovin** [reply]
Totally makes sense, thanks for the extra context.

**Comment by neonflame4** [comment]
No problem! Thanks for the response!
